### ISSUE TITLE
Mobile header pulldown toggle has no text alternative

### ISSUE THREAD

**Issue by cjg89** [UCF/Pegasus-Theme#114]
https://github.com/UCF/Pegasus-Theme/blob/master/versions/v6/header.php#L67

This link needs some unique inner text, like "Toggle Menu".

This issue is present in all versions, but should be easy to address in all of them, as the markup is basically the same:

https://github.com/UCF/Pegasus-Theme/blob/master/versions/v5/header.php#L61
https://github.com/UCF/Pegasus-Theme/blob/master/versions/v4/header.php#L61
https://github.com/UCF/Pegasus-Theme/blob/master/versions/v3/header.php#L106
https://github.com/UCF/Pegasus-Theme/blob/master/versions/v2/header.php#L105
https://github.com/UCF/Pegasus-Theme/blob/master/versions/v1/header.php#L105
